src/connectors/scenes/TransactionListConnector.js

```diff
@@ -77,7 +85,8 @@ const mapStateToProps = (state: State) => {
     showToWalletModal: state.ui.scenes.walletListModal.walletListModalVisible,
     requiredConfirmations,
     numTransactions,
-    isBalanceVisible
+    isBalanceVisible,
+    loadingPercent
```

Comment:
Do not connect `loadingPercent` directly to redux using the Scene's connector. This will cause a re-render of the entire scene on each loadingPercent update.

Please use the WiredProgressBar like it was used in the WalletListScene